src/content/docs/ja/guides/upgrade-to/v5.mdx

```diff
@@ -0,0 +1,1219 @@
+
+この動作を無効にするには、明示的に `security.checkOrigin: false` を設定する必要があります。
+
+```js title="astro.config.mjs" ins={3-5}
+export default defineConfig({
+  output: "server",
+  security: {
+    checkOrigin: false
+  }
+})
+```
+
+<ReadMore> [セキュリティ設定オプション (EN)](/en/reference/configuration-reference/#security)について詳しくはこちらをご覧ください。</ReadMore>。
+
+### 注入されたルートとリダイレクトのルート優先順位
+
+<SourcePR number="11798" title="従来のルート優先順位付けの削除(EN)"/>
+
+Astro v4.xでは、`experimental.globalRoutePriority` は、注入されたルート、ファイルベースのルート、およびリダイレクトがすべて [すべてのルートのルート優先順位ルール](https://www.google.com/url?sa=E&source=gmail&q=/ja/guides/routing/#ルーティングの優先順位) を使用して優先順位付けされるようにするオプションのフラグでした。これにより、特定の種類のルートを自動的に優先順位付けせず、ルート優先順位を標準化することで、プロジェクトのルーティングをより細かく制御できました。
+
+Astro v5.0では、この実験的なフラグが削除され、Astro の新しいデフォルトの動作になりました。リダイレクトと注入されたルートは、ファイルベースのプロジェクトルートと同様に優先順位付けされるようになりました。
+
+これは Starlight のデフォルトの動作であり、更新された Starlight プロジェクトには影響しません。
+
+#### どうすればいいですか？
+
+プロジェクトに注入されたルートまたはリダイレクトが含まれている場合は、ルートが期待どおりにページURLを構築していることを確認してください。新しい動作の例を以下に示します。
+
+次のルートを含むプロジェクトの場合、
+
+- ファイルベースのルート: `/blog/post/[pid]`
+- ファイルベースのルート: `/[page]`
+- 注入されたルート: `/blog/[...slug]`
+- リダイレクト: `/blog/tags/[tag] -> /[tag]`
+- リダイレクト: `/posts -> /blog`
+
+以下のURLは、Astrov4.xのルート優先順位に従わずに構築されます。
+
+- `/blog/tags/astro` は、リダイレクトによって `/tags/[tag]`に構築されます (注入されたルート `/blog/[...slug]` ではなく)
+- `/blog/post/0` は、ファイルベースのルート `/blog/post/[pid]`によって構築されます (注入されたルート `/blog/[...slug]` ではなく)
+- `/posts` は、リダイレクトによって `/blog`に構築されます (ファイルベースのルート `/[page]` ではなく)
+
+ルートの衝突が発生した場合、つまり、同じルート優先順位の 2 つのルートが同じ URL を構築しようとした場合、Astro は競合するルートを識別する警告をログに記録します。
+
+<ReadMore> [ルーティングの優先順位](https://www.google.com/url?sa=E&source=gmail&q=/ja/guides/routing/#ルーティングの優先順位)について詳しくはこちらをご覧ください。</ReadMore>
+
+### `<script>`タグは宣言どおりに直接レンダリングされるようになりました
+
+<SourcePR number="11791" title="`directRenderScript`をデフォルトにする(EN)"/>
+
+Astro v4.xでは、`experimental.directRenderScript`は、`.astro`ファイルで宣言された`<scripts>`を直接レンダリングするためのオプションのフラグでした(TypeScript、`node_modules` のインポート、スクリプトの重複排除などの既存の機能を含む)。この戦略により、スクリプトが使用されていない場所で実行されるのを防ぎました。さらに、条件付きでレンダリングされたスクリプトは、`is:inline`ディレクティブが自動的に追加されたかのように、暗黙的にインライン化されていました。
+
+Astro 5.0では、この実験的なフラグが削除され、Astro の新しいデフォルトの動作になりました。スクリプトは `<head>`にホイストされなくなり、ページ上の複数のスクリプトはバンドルされなくなり、`<script>` タグが CSS スタイルに干渉する可能性があります。さらに、条件付きでレンダリングされたスクリプトは暗黙的にインライン化されなくなりました。
+
+#### どうすればいいですか？
+
+`<script>` タグを確認し、期待どおりに動作することを確認してください。
+
+以前に条件付きでレンダリングされた `<script>` タグがあった場合は、以前と同じ動作を維持するために `is:inline` 属性を追加する必要があります。
+
+```astro title="src/components/MyComponent.astro" ins="is:inline"
+---
+type Props = {
+  showAlert: boolean
+}
+
+const { showAlert } = Astro.props;
+---
+{
+  showAlert && <script is:inline>alert("Some very important code!!")</script>
+}
+```
+
+<ReadMore> [Astro で `script` タグを使用する]([https://www.google.com/url?sa=E&source=gmail&q=/ja/guides/client-side-scripts/#astroでscriptを使用する](https://www.google.com/url?sa=E&source=gmail&q=/ja/guides/client-side-scripts/#astroでscriptを使用する))について詳しくはこちらをご覧ください。</ReadMore>
```

Comment:
Hey @n-tong009, could you take a look at those broken links here? I believe you might've pasted them wrong or something, since they aren't supposed to start with ths Google prefix.